pallets/msa/src/lib.rs

```diff
@@ -244,6 +245,13 @@ pub mod pallet {
 	#[pallet::whitelist_storage]
 	pub(super) type OffchainIndexEventCount<T: Config> = StorageValue<_, u16, ValueQuery>;
 
+	/// Storage type for mapping MSA IDs to their Recovery Commitments
+	/// - Key: MessageSourceId
+	/// - Value: Recovery Commitment (32 bytes)
+	#[pallet::storage]
+	pub type MsaIdToRecoveryCommitment<T: Config> =
+		StorageMap<_, Twox64Concat, MessageSourceId, [u8; 32], OptionQuery>;
```

Comment:
nit: We created a type alias for `RecoveryCommitment = [u8; 32]`. Can we use that here?